cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -462,6 +520,20 @@ mod test {
         }
     }
 
+    fn create_price_update_msg(emitter_address: &[u8], emitter_chain: u16) -> Binary {
+        let batch_attestation = BatchPriceAttestation {
+            // TODO: pass these in
+            price_attestations: vec![],
```

Comment:
we'll be able to use this in the price update tests as well, but i'll do it separately because it'll touch a lot of code.